src/openslide-vendor-dicom.c

```diff
@@ -442,24 +431,14 @@ static bool read_tile(openslide_t *osr,
   debug("read_tile level:");
   print_level(l);
 
-  int64_t frame_index = tile_col + l->tiles_across * tile_row;
```

Comment:
This is the last user of `l->tiles_across`, and `l->tiles_down` already didn't have any, so we could drop those fields.

Reply:
Oh yeah, nice.